src/BannedSymbols.txt

```diff
@@ -0,0 +1 @@
+M:System.Globalization.CompareInfo.IndexOf(System.String,System.Char)
```

Comment:
I'm assuming this can't have a copyright banner.

Reply:
Roslyn's don't https://github.com/dotnet/roslyn/blob/7fd079ef9d14d04978caaee4baa67d97273cb01d/src/CodeStyle/BannedSymbols.txt

And it doesn't look like the parser supports it

https://github.com/dotnet/roslyn-analyzers/blob/f64768fabfba9eb8d6569a90560b9937bb489cc7/src/Microsoft.CodeAnalysis.BannedApiAnalyzers/Core/SymbolIsBannedAnalyzer.cs#L182-L192

So going with "no".